actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -37,7 +37,8 @@ module TranslationHelper
       # you know what kind of output to expect when you call translate in a template.
       def translate(key, options = {})
         options = options.dup
-        options[:default] = wrap_translate_defaults(options[:default]) if options[:default]
+        remaining_defaults = Array(options.delete(:default))
+        options[:default] = remaining_defaults.shift if remaining_defaults.first.kind_of? String
```

Comment:
Again, it is not being restricted to strings. See https://github.com/rails/rails/pull/18308/files#diff-59dd14805efd3bccf48e4f2e283880e9R131